spicedb/docker-compose.yaml

```diff
@@ -0,0 +1,62 @@
+---
+# This runs SpiceDB, using Postgres as the storage engine. SpiceDB will not have
+# any schema written to it.
+#
+# Once the database service is running, the migrate service executes, running
+# "spicedb migrate head" to migrate Postgres to the most current revision. After
+# Postgres is migrated, the migrate service will stop.
+#
+# SpiceDB settings:
+#   pre-shared key: foobar
+#   dashboard address: http://localhost:8080
+#   metrics address: http://localhost:9090
+#   grpc address: http://localhost:50051
+#
+# Postgres settings:
+#   user: postgres
+#   password: secret
+#   port: 5432
+
+version: "3"
+
+services:
+  rebac:
```

Comment:
I might have confused things yesterday when I said I would like to bring up the rebac service with docker-compose. 

I was thinking we would run: `docker-compose up --build` to build the image dynamically, rather than pull it from quay.io. It seems like you don't need any reference to the service at all to do this, e.g. https://github.com/merlante/inventory_access_poc/blob/main/docker-compose.yml

Reply:
https://github.com/RedHatInsights/insights-rebac/pull/15